tests/test_pesq.py

```diff
@@ -1,9 +1,12 @@
-from pathlib import Path
+#!/usr/bin/python3
```

Comment:
When it is necessary to add something like this, could you use `#!/usr/bin/env python3`?

Reply:
I actually thought I had removed it. I have done it now. I'll commit it alongside with whatever changes may still be required.